Restfull API/api/app/models.py

```diff
@@ -0,0 +1,33 @@
+from flask import session, request
```

Comment:
NOTE: You are building an API, why do you have sessions? API should not have session data.

Reply:
Note: i should look at jwt